### DISCUSSION TITLE
How to read the output of NFA (Nemo Forced Aligner Tool) ?

### DISCUSSION THREAD

**Discussion by Ca-ressemble-a-du-fake** [NVIDIA-NeMo/NeMo#5848] [Q&A] [answered]
Hi,

I tried NFA to align a sentence with audio thanks to citrinet fr version. Yet I don't fully understand the output : 

> \<utt_id> 1 \<start time in samples> \<duration in samples> \<text, ie token/word/segment>

Duration in samples is given as a decimal value like `19.52`. I would expect it to be an integer representing the number of samples at a sampling rate of 16kHz. But this is not the case.

So how to convert this duration in samples to a duration in seconds ?

Thanks in advance for your answer,

**Comment by titu1994** [comment, marked as answer]
According to @erastorgueva-nv 
The float values represents the time in seconds.

**Reply by Ca-ressemble-a-du-fake** [reply]
Thanks I'll take note of this.
